josm-presets/de-signals-bostrab.xml

```diff
@@ -576,6 +576,38 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 			</item>
 		</group>
 		<group name="Electricity Signals (St)" de.name="Schaltsignale (St)">
+			<item name="Signalcontact (St 1)" de.name="Signalkontakt (St 1)" type="node">
+				<label text="Signalcontact (St 3)" de.text="Signalkontakt (St 1)" />
+				<label de.text="Wird als Punkt auf dem Gleis erfasst." text="Is being mapped as a node on the track." />
+				<space />
+				<key key="railway"
+					value="signal" />
+				<check key="railway:signal:electricity:deactivated"
+					text="Out of order"
+					de.text="Ungültigkeitskreuz"
+					/>
+				<combo key="railway:signal:position"
+					text="Signal position"
+					de.text="Signalstandort"
+					values="left,right,overhead"
+					de.display_values="Links von Wegrichtung,Rechts von Wegrichtung,Oberleitung"
+					display_values="left,right,catenary"
+					/>
+				<reference ref="direction_fw_bw" />
+				<key key="railway:signal:electricity"
+					value="DE-BOStrab:st1" />
```

Comment:
Although the signal is part of the St group, it is not a signal related to electricity for traction but to control points. `railway:signal:switch=DE-BOStrab:st1` looks better because it is about points (American English: switches).

Please document the tagging on the wiki. https://wiki.openstreetmap.org/wiki/DE:OpenRailwayMap/Tagging_Trams_in_Germany#Signalkontakt_.28St_1.29 shows how it looks but it does not describe its tagging.